File: sdk/python-tunzisha/get_all_patient_data.py
import requests

url = "https://3000-kcelestinom-tunzishaxch-gy1ysd1zmzc.ws-eu108.gitpod.io/api/Patients"

payload = ""
headers = {}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)

File: sdk/java-tunzisha/get_all_patient_data.java
public class get_all_patient_data {

    public static void main(String[] args) {

        OkHttpClient client = new OkHttpClient().newBuilder()
                .build();
        MediaType mediaType = MediaType.parse("text/plain");
        RequestBody body = RequestBody.create(mediaType, "");
        Request request = new Request.Builder()
                .url("https://localhost:3000/api/Patients")
                .method("GET", body)
                .build();
        Response response = client.newCall(request).execute();
    }
}
